sqllineage/core/handlers/source.py

```diff
@@ -74,6 +74,9 @@ def _handle_table(self, token: Token, holder: SubQueryLineageHolder) -> None:
                 self._handle(token.tokens[1], holder)
         elif token.ttype == Literal.String.Single:
             self.tables.append(Path(token.value))
+        # If we have a CREATE TABLE AS SELECT * FROM VALUES (...), this stops the parser erroring out:
+        elif type(token).__name__ == "Token" and str(token)[0:6].lower() == "values":
+            pass
```

Comment:
The token here is a instance of Parenthesis. Please reuse that elif above. Also, make the function call pythonic. Don't use __name__ or str casting when we have alternative choice.

Reply:
Thank you for your great feedback, it really helps me to learn too!

I have tried to use the Parenthesis IF [from here](https://github.com/reata/sqllineage/blob/e67293a874e34f5d11baa88aa47b3aa7af99f1e4/sqllineage/core/handlers/source.py#L58), but I get failed tests if I do this:
```python
        elif isinstance(token, Parenthesis):
            if is_subquery(token):
                # SELECT col1 FROM (SELECT col2 FROM tab1), the subquery will be parsed as Parenthesis
                # This syntax without alias for subquery is invalid in MySQL, while valid for SparkSQL
                self.tables.append(SubQuery.of(token, None))
            elif str(token)[0:6].lower() == "values":
                # If we have a CREATE TABLE AS SELECT * FROM VALUES (...), this stops the parser erroring out:
                pass
            else:
                # SELECT * FROM (tab2), which is valid syntax
                self._handle(token.tokens[1], holder)
        elif token.ttype == Literal.String.Single:
            self.tables.append(Path(token.value))
        else:
            raise SQLLineageException(
                "An Identifier is expected, got %s[value: %s] instead."
                % (type(token).__name__, token)
            )
```

failed test: 
```
FAILED tests/test_others.py::test_view_as_select_from_values_with_colnames - sqllineage.exceptions.SQLLineageException: An Identifier is expected, got Token[value: VALUES] instead.
FAILED tests/test_others.py::test_view_as_select_from_values_no_colnames - sqllineage.exceptions.SQLLineageException: An Identifier is expected, got Token[value: VALUES] instead.
```

Sorry if I'm missing something obvious, I haven't seen these custom subtypes used before and I don't yet know how to run SQL against your library and see which one an input gets detected as.

Reply:
Yes this is the right place. You can look at how we decide a Parenthesis is subquery or not. Basically it's finding the first token and see whether it's SELECT. Same here, we'd better fill in a logic to make sure the first token within Parenthesis is keyword VALUES, only after that can be sure that this can be skipped. 

As for 'how to run SQL against your library and see which one an input gets detected as', you can use an debugger or simply add a print statement. The token type and token class come from sqlparse library. You can also refer to its document.

Reply:
I switched to a Pythonic token test as per the comment in your other review. Thanks for the help!